tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/files/StorageSpec.scala

```diff
@@ -84,41 +68,41 @@ abstract class StorageSpec extends BaseIntegrationSpec {
   "An empty file" should {
 
     "be successfully uploaded" in {
-      uploadFile(emptyTextFile, None)(expectCreated)
+      filesDsl.uploadFile(emptyTextFile, projectRef, storageId, None)(expectCreated)
     }
 
     "be downloaded" in {
       deltaClient.get[ByteString](s"/files/$projectRef/attachment:empty", Coyote, acceptAll) {
-        expectDownload("empty", ContentTypes.`text/plain(UTF-8)`, emptyFileContent)
+        filesDsl.expectDownload("empty", ContentTypes.`text/plain(UTF-8)`, emptyFileContent)
       }
     }
   }
 
   "A json file" should {
 
     "be uploaded" in {
-      uploadFile(jsonFileNoContentType, None)(expectCreated)
+      filesDsl.uploadFile(jsonFileNoContentType, projectRef, storageId, None)(expectCreated)
     }
 
     "be downloaded" in {
       deltaClient.get[ByteString](s"/files/$projectRef/attachment:attachment.json", Coyote, acceptAll) {
-        expectDownload("attachment.json", ContentTypes.`application/json`, jsonFileContent)
+        filesDsl.expectDownload("attachment.json", ContentTypes.`application/json`, jsonFileContent)
```

Comment:
I realise the name is the same as before, but `expectDownload` isn't great, and the hiding of the response and headers isn't helpful to the reader